clusterloader2/pkg/flags/types.go

```diff
@@ -17,9 +17,9 @@ limitations under the License.
 package flags
 
 import (
-	"strconv"
-
 	"github.com/spf13/pflag"
+	"strconv"
```

Comment:
Please revert - the convention is that the first group of important are builtin go libraries, exactly how it was before.

Reply:
Outdated